medusa/indexers/tvdbv2/tvdbv2_api.py

```diff
@@ -33,6 +34,53 @@
 
 logger = logging.getLogger(__name__)
 
+FALLBACK_TIMEOUT = 600
+API_BASE_TVDB = 'https://api.thetvdb.com'
+API_BASE_URL_FALLBACK = 'https://tvdb2.plex.tv'
+# Initiate the tvdb api v2
+api_base_url = API_BASE_TVDB
+plex_fallback_time = datetime.datetime.now()
+tvdb_connect_failures_count = 0
+
+
+def plex_fallback(func):
+    def inner(*args, **kwargs):
+        global plex_fallback_time
+        global api_base_url
+        global tvdb_connect_failures_count
+
+        if plex_fallback_time + datetime.timedelta(seconds=FALLBACK_TIMEOUT) < datetime.datetime.now():
+            api_base_url = API_BASE_TVDB
+            tvdb_connect_failures_count = 0
+        else:
+            logger.debug("Plex fallback still enabled.")
+
+        try:
+            api_result = func(*args, **kwargs)
+        except ApiException as e:
+            logger.warning("could not connect to TheTvdb.com, reason '%s',%s)", e.reason, e.status)
+            tvdb_connect_failures_count += 1
+            if tvdb_connect_failures_count < 4:
+                raise
+        except (MaxRetryError, RequestError) as e:
+            logger.warning("could not connect to TheTvdb.com, with reason '%s'.", e.reason)
+            tvdb_connect_failures_count += 1
+            if tvdb_connect_failures_count < 4:
+                raise
+        except Exception as e:
+            logger.warning("could not connect to TheTvdb.com, with reason '%s'.", e)
```

Comment:
If you change the apikey, it will get to this, through a 401 exception. And you can (kinda) test it.